src/components/v2/Modal/ModalStyles.ts

```diff
@@ -0,0 +1,61 @@
+import { useTheme } from '@mui/material';
+
+const iconCloseSize = '35px';
+
+export const useModalStyles = ({
+  hasTitleComponent,
+  noHorizontalPadding,
+}: {
+  hasTitleComponent: boolean;
+  noHorizontalPadding?: boolean;
+}) => {
+  const theme = useTheme();
+  return {
+    box: {
+      position: 'absolute' as const,
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
+      width: '80%',
+      maxWidth: 544,
+      boxShadow: 24,
+      padding: 0,
+      borderRadius: theme.spacing(3),
+      paddingTop: theme.spacing(3),
+    },
+    titleWrapper: {
+      position: 'relative',
+      paddingLeft: theme.spacing(3),
+      paddingRight: theme.spacing(3),
+      paddingBottom: hasTitleComponent ? theme.spacing(3) : 0,
+      borderBottom: hasTitleComponent ? `1px solid ${theme.palette.secondary.dark}` : 0,
```

Comment:
```suggestion
      borderBottom: hasTitleComponent ? `1px solid ${theme.palette.secondary.light}` : 0,
```

<img width="725" alt="Screenshot 2022-03-25 at 13 02 00" src="https://user-images.githubusercontent.com/44675210/160125656-dae3aec7-1b26-4410-b455-73298316f3a4.png">